src/main/webapp/app/shared/modal/RelevantCancerTypesModal.tsx

```diff
@@ -212,14 +214,20 @@ const RelevantCancerTypesModalContent = observer(
         </ModalBody>
         <ModalFooter style={{ display: 'inline-block' }}>
           <div className="d-flex justify-content-between">
-            {allCancerTypesDeleted ? (
+            {allCancerTypesDeleted && (
               <div className="error-message">
                 <FaExclamationCircle className="mr-2" size={'25px'} />
                 <span>You must include at least one cancer type</span>
               </div>
-            ) : (
-              <div />
             )}
+            {numberDeletedRcts > 0 && !allCancerTypesDeleted && (
+              <div className="d-flex align-items-center">
+                <span>
+                  {numberDeletedRcts === 1 ? `There is 1 excluded cancer type` : `There are ${numberDeletedRcts} excluded cancer types`}
```

Comment:
We have a library called `pluralize` that you can use here to use the plural version of `is` and `type` based on number.